test/devel_env.sh

```
@@ -0,0 +1,275 @@
+
+sleep 2
+
+if ! kill -0 "$DEVEL_LSMD_PID" 2>/dev/null; then
+    DEVEL_LSMD_PID=""
+fi
+if [ -z "$DEVEL_LSMD_PID" ]; then
+    echo "ERROR: Failed to start lsmd daemon."
+    echo "       Check $DEVEL_BASE/logs/lsmd.log"
+    cat "$DEVEL_BASE/logs/lsmd.log" 2>/dev/null
+    unset _devel_clib_so _DEVEL_SCRIPT_DIR _DEVEL_SRC_DIR _DEVEL_BUILD_DIR _DEVEL_LIBTOOL
+    return 1 2>/dev/null || exit 1
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)